MailCore/Utils/Constants.swift

```diff
@@ -96,6 +96,7 @@ public enum Constants {
 
     public static let mailRegex =
         "(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|\"(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21\\x23-\\x5b\\x5d-\\x7f]|\\\\[\\x01-\\x09\\x0b\\x0c\\x0e-\\x7f])*\")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\\x01-\\x08\\x0b\\x0c\\x0e-\\x1f\\x21-\\x5a\\x53-\\x7f]|\\\\[\\x01-\\x09\\x0b\\x0c\\x0e-\\x7f])+)\\])"
+    public static let referenceRegex = ">\\s*<|>?\\s+<?"
```

Comment:
Compute the NSRegularExpression only once (and remove ...using(regex:) where needed)
```
    public static var referenceRegex: NSRegularExpression = {
        // It's safe to unwrap because this will always succeed
        // swiftlint:disable force_try
        return try! NSRegularExpression(pattern: ">\\s*<|>?\\s+<?")
    }()
    ```